.eslintcache

```diff
@@ -0,0 +1 @@
sonal/ember-lifeline/addon/debounce-task.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/dom-event-listeners.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/index.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/poll-task.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/run-task.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/types/index.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon/utils/get-task.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon-test-support/index.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon-test-support/poll-task-for.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/addon-test-support/setup-lifeline-validation.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/config/addon-docs.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/config/deploy.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/config/ember-try.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/config/environment.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/ember-cli-build.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/index.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/testem.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/acceptance/cleanup-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/app.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/components/run-task-demo.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/config/environment.d.ts",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/resolver.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/router.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/routes/foo.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/config/environment.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/config/targets.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/destroy-app.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/module-for-acceptance.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/resolver.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/start-app.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/test-helper.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/debounce-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/dom-event-listeners-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/poll-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/run-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/utils/get-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/types/dummy/index.d.ts",[]]
```

Comment:
This should not be committed (should be added to `.gitignore`)

Reply:
Ya not sure why this snuck in.

Reply:
Fixed.